jenkins/ci.suse.de/templates/cloud-mkcloud-gating-template.yaml

```diff
@@ -44,4 +44,3 @@
           predefined-parameters: |
             project=Devel:Cloud:{version}
             subproject=Staging
-            packagefilter=crowbar python openstack patterns release-notes-suse-openstack-cloud suse-cloud-upgrade
```

Comment:
Why did we use the filter before? Disk space? Network bandwidth?

Reply:
It was still from very old times where we didn't use a CI for this and the staging purpose was very different back then. Nowadays we use it for nearly everything.

Reply:
Thanks, that makes sense then.

Reply:
FYI: The filter was meant to selectively submit packages to the non-staging project. This selection is no longer made and the hard coding of all packages renders the filter useless.
Therefore, the new default is: if the filter is empty, handle all packages, if the filter is defined, do the filtering.